src/v/config/configuration.cc

```diff
@@ -3916,6 +3916,41 @@ configuration::configuration()
       std::chrono::milliseconds{1min},
       {.min = std::chrono::milliseconds{10s},
        .max = serde::max_serializable_ms})
+  , datalake_scheduler_block_size_bytes(
+      *this,
+      "datalake_scheduler_block_size_bytes",
+      "Size, in bytes, of a memory block reserved for record translation, as "
+      "tracked by the datalake scheduler.",
+      {.needs_restart = needs_restart::yes, .visibility = visibility::tunable},
+      4_MiB,
+      {.min = 1_MiB, .max = 8_MiB})
+  , datalake_scheduler_max_concurrent_translations(
+      *this,
+      "datalake_scheduler_max_concurrent_translations",
+      "The maximum number of translations that the datalake scheduler will "
+      "allow to run at a given time. If a translation is requested but the "
+      "number of running translations exceeds this value, the request will be "
+      "put to sleep temporarily, polling until capacity becomes available.",
+      {.needs_restart = needs_restart::yes, .visibility = visibility::tunable},
+      4,
+      {.min = 1, .max = 8})
+  , datalake_scheduler_time_slice_ms(
+      *this,
+      "datalake_scheduler_time_slice_ms",
+      "Time given, in milliseconds, to a datalake translation as scheduled by "
+      "the datalake scheduler. Once a translation is scheduled, it will run "
+      "until either a) the time slice has elapsed or b) all pending records "
+      "on its souirce partition have been translated.",
+      {.needs_restart = needs_restart::yes, .visibility = visibility::tunable},
+      30s,
+      {.min = 1s, .max = 60s})
+  , iceberg_data_path_prefix(
+      *this,
+      "iceberg_data_path_prefix",
+      "Path prefix for uploading parquet files to remote object storage. "
+      " TODO: is it safe to set this after iceberg has been switched on?",
```

Comment:
What's this for? Why not just hardcode `data` and `metadata`? Should we have one for metadata too?

>TODO: is it safe to set this after iceberg has been switched on

Is there a specific concern? It doesn't smell particularly unsafe

Reply:
> What's this for?

Added it in response to a TODO in Bharath's PR that a block of static values should/could be configurable. I don't see an obvious reason for this one, so it might have been lumped in by mistake.

> specific concern?

Not really. I see a remote path, don't fully understand the implications of changing it on a live system, so I have to ask. Probably we can just remove this config though. @bharathv?

Reply:
ya we can remove.. I didn't mean to make it a configurable (perhaps the code just got lumped under a TODO that caused this confusion, apologies)

Reply:
done